hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -196,34 +197,52 @@ public Pair<ResponseCodeEnum, Long> create(
                     "Cannot auto-create an account from unaliased change " + change);
         }
 
-        // checks tokenAliasMap if the change consists an alias that is already used in previous
-        // iteration of the token transfer list. This map is used to count number of
-        // maxAutoAssociations needed on auto created account
-        analyzeTokenTransferCreations(changes);
+        TransactionBody.Builder syntheticCreation;
+        HederaAccountCustomizer customizer;
+        if (alias.size() == EntityIdUtils.EVM_ADDRESS_SIZE) {
+            syntheticCreation = syntheticTxnFactory.createHollowAccount(alias, 0L);
+            customizer =
+                    new HederaAccountCustomizer()
+                            .memo(AUTO_MEMO)
```

Comment:
nit: use a lazy create memo
```
public static final String LAZY_MEMO = "lazy-created account";
```
```suggestion
                            .memo(LAZY_MEMO)
```